PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Description
+
+Describe the the problem this PR solves. Add a link to the ticket if any.
+
+### Proposed changes
+
+Please explain the changes you made here. Provide a video or screenshot if nessessary.
+
+### Checklist
+
+- [ ] Lint Rules are passed correctly
+- [ ] Code builds correctly
+- [ ] Created or updated tests for the new code
+- [ ] All existing and new tests passed
```

Comment:
Personally i think that just the 3rd point is something what we need here. The other ones are already "checked" by our automations. I think we do not need a manual checkbox where we already have an automation for. :thinking:

Reply:
Also, not all projects have eslint or sonar integrated, right?

Reply:
I agree with you @rcidaleassumpo, we can't trust anyone 😄

However, we could start with the `Created or updated tests for the new code` check and see if people are more aware of writing test cases

Reply:
I like the compromise to start with just the one entry in the checklist.